static/src/javascripts/projects/common/modules/commercial/third-party-tags.js

```diff
@@ -63,6 +63,7 @@ define([
         remarketing.load();
         outbrain.load();
         krux.load();
+        gravity.getRecommendations();
```

Comment:
I've just merged deferring third parties so this will probably go to the loadOther function then as it should be loaded after ads.